service/frontend/workflowHandler.go

```diff
@@ -2093,6 +2093,17 @@ func (wh *WorkflowHandler) ResetWorkflowExecution(ctx context.Context, request *
 		return nil, err
 	}
 
+	switch request.GetResetReapplyType() {
```

Comment:
I moved all default to `enums/defaults.go` file.

Reply:
Is this just try to set a default option if unspecified? If that is the intention, just do if check would be shorter and clearer. 
if request.GetResetReapplyType() == enumspb.RESET_REAPPLY_TYPE_UNSPECIFIED {
  request.ResetReapplyType = enumspb.RESET_REAPPLY_TYPE_ALL
}